angular/projects/spark-angular/src/lib/components/sprk-promo/sprk-promo.stories.ts

```diff
@@ -61,17 +84,27 @@ defaultStory.story = {
 export const flag = () => ({
   moduleMetadata: modules,
   template: `
-    <sprk-promo
-      isFlag="true"
-      imgSrc="https://spark-assets.netlify.app/spark-logo-updated.svg"
-      imgAlt="Spark Design System Logo"
-      imgHref="#"
-      idString="promo-2"
-      hasBorder="true"
-    >
-      Lorem ipsum dolor. Sit amet pede. Tempus donec et. Suspendisse id
-      inventore integer eum non enim diam habitant. Maecenas nunc per lacus
-      neque egestas. Diam quod curabitur.
+    <sprk-promo idString="flag-promo" hasBorder="true" isFlag="true" additionalClasses="sprk-o-Stack--split@s">
+      <a href="#nogo" sprkStackItem class="sprk-o-Stack__item--fourth@s">
+        <img
+          sprkPromoMedia
+          isFlag="true"
+          alt="Spark Design System Logo."
```

Comment:
When an image is the only content inside a link, then the alt text should describe the behavior of the link. In this case, something like "Learn more about the Spark Design System".